src/coreComponents/mesh/generators/InternalWellboreGenerator.cpp

```diff
@@ -346,6 +346,10 @@ void InternalWellboreGenerator::coordinateTransformation( arrayView2d< real64, n
   SortedArray< localIndex > & tnegNodes = nodeSets["tneg"];
   SortedArray< localIndex > & tposNodes = nodeSets["tpos"];
 
+  // Nodesets for the casing-cement and cement-rock interfaces of a cased wellbore
+  SortedArray< localIndex > & rCasingCementInterfaceNodes = nodeSets["rCasingCementInterface"];
```

Comment:
to me it's a little unclear why these nodesets need to be predefined. They kind of mix geometry and physics. what if one had a wellbore like problem geometry but not regions called casing, cement and rock? I feel these sets should be defined using geometric objects or directly in the mesh through a `vtk` mesh.

Reply:
Yes, you're absolutely right.
Actually I saw it and I was like "_Let it go_", but I should not have.
Thanks for double checking it.

What do you think a proper name would be? `InternalInterface`, `ExternalInterface`?

Reply:
those names certainly sound a lot better to me. However, there is no reason for always having 3 `cellblocks` so I would limit myself to a set defining the internal surface if doesn't already exist. interfaces between regions (or cellblocks) should not have predefined sets IMO. In a case with a single region those sets would become meaningless. 

It should be easy enough to create the set using a cylinder.

Reply:
If a hollow cylinder does not exist yet, then we should create it and use it to create these nodesets

Reply:
> It should be easy enough to create the set using a cylinder.

@CusiniM I like the idea of create nodeset within `Geometry` using a cylinder. It will be more generic. I will try with this idea.

Reply:
@CusiniM @TotoGaz @herve-gross Following the suggestion of Mateo, I tested the `Cylinder` geometry. It works perfectly for this purpose. So I will close this PR. However I will propose some minor changes in another PR to clean the `Cylinder` class.